content/en/docs/papers/fsep/_index.md

```diff
@@ -0,0 +1,218 @@
+- Give the users the ability to vet incoming connection requests to validate that they are not from problematic sites. 
+- Expand blocklist management by listing why a site is blocked, access to available examples, and when the site was last updated.  
+
+While adding these features will enhance the usability of blocklists and upgrade the end-user experience, corresponding front-end changes are required in several areas to ask permission for these services to be used and to make how those services are being used clear and easy to understand.
+
+### Onboarding
+
+The administration onboarding process can be enhanced by leveraging the additions to the back end detailed in the Service Feature Integration. 
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 1.1 | At setup time, administrators should be able to upload a deny list or select a default to deny list provided. For MVP, only one deny list provider needs to be supported in this list (The Bad Space). Ultimately, the goal is to allow users to choose from one of several vetted tier 0 lists. | P1 |
+
+| 1.2 | At setup time, administrators should be able to select whether or not they want the default deny list to auto-update. If the admin selects “yes,” the list should attempt to auto-update once every 24 hrs.  | P1 |
+
+### Deny List Management
+
+Deny list management will be extended with new features to provide more information and nuanced options to curate a specific experience. 
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 2.1 | The administration panel should have a new option to manage “deny lists” and blocked sites. This list should contain all deny lists that the admin has subscribed to. | P1 |
+| 2.2 | Auto updates: It should be possible to have the system check for deny list subscription updates at regular intervals, i.e., once every 24 hours. Local status overrides will take precedence over imported instance statuses. | P1 |
+| 2.3 | Check for updates: It should be possible for an admin to manually update any of the deny lists that they are subscribed to outside of the automated update time. | P1 |
+| 2.4 | Overrides: It should be possible for an admin to add a manual override for a deny list. The manual override should be applied over all deny lists. | P1 |
+| 2.5 | Remove Instance from deny list: It should be possible for an administrator to remove instances where deemed appropriate. These changes only apply to local deny lists and will not affect subscribed deny lists.  | P1 |
+
+### Moderation Data
+
+Moderation capabilities will be expanded to provide more context and information to make informed decisions. 
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 3.1 | It should be possible for admins to see how many of their users’ followers / following would be impacted by a deny list. For example: *“When you subscribe to [The Bad Space], [Number] of your users will no longer be able to communicate with [number] followers and [number] following.”*| P1 |
```

Comment:
There's a new API in Mastodon 4.2 that calculates this, though I'm not sure if it's a public-use API or something internal to the admin panel.

Reply:
Cool.